app/src/main/kotlin/org/eurofurence/connavigator/ui/FragmentViewAbout.kt

```diff
@@ -42,7 +56,7 @@ class FragmentViewAbout : Fragment(), ContentAPI {
         ui.requinardLayout.setOnClickListener { browse("https://furry.requinard.nl") }
         ui.pazuzuLayout.setOnClickListener { browse("https://twitter.com/Pazuzupizza") }
 
-        ui.helpButton.setOnClickListener { browse("https://t.me/joinchat/AHbDIgZX0NYGill2_ikXXA") }
+        ui.helpButton.setOnClickListener { browse(RemotePreferences.supportChatUrl) }
         ui.bugButton.setOnClickListener { browse("https://github.com/eurofurence/ef-app_android/issues") }
```

Comment:
I love the idea of putting that into a remote config, too... if it's not too much work.